### PR TITLE
POL-217: Add Syntax examples for Condition builder

### PR DESCRIPTION
**Pull request by spinder**
I'm not getting the final spacing around the StackItem 'right', too much space, or able to figure out how to get the drop down flag widget moved to the left of the accordian display.  
Screenshot2020-04-17_17-57-12
Screenshot2020-04-17_17-56-35

### CODE REVIEW

**Comment by josejulio** [comment]
I see that you used the [accordion](https://www.patternfly.org/v4/documentation/react/components/accordion) component, but the [Expandable](https://www.patternfly.org/v4/documentation/react/components/expandable)  (I think you can use the Uncontrolled one) matches design better.

There are conflicts now

**Comment by spinder** [comment]
Screenshot2020-04-17_22-45-22
Screenshot2020-04-17_22-44-33

0) Rebased to get rid of merge errors.
1)Used the new component and removed all the other code related to the previous component and state tracking.
2)The spacing inside the Expandable is bigger than I thought it would be. Suggestions? Should we put a small gray box around it when expanded? The issue is that fully expanded the conditions box is pushed off the screen.  Go still smaller on the text size? Decrease spacing in between lines?

**Comment by josejulio** [comment]
I think we need to put a left padding ([see mock up](https://user-images.githubusercontent.com/1906187/79626334-618e0500-80fd-11ea-9cd4-e9f9207d8d40.png)) of the expanded content (but not the expandable link it self) to align it properly

> 1. 2)The spacing inside the Expandable is bigger than I thought it would be. Suggestions? Should we put a small gray box around it when expanded? The issue is that fully expanded the conditions box is pushed off the screen.  Go still smaller on the text size? Decrease spacing in between lines?

The spacing inside seems too much for me, in the design looks good because the space is too big  there :)

@katierik what do you suggest? should we pull the condition text above the text or?

**Comment by pilhuhn** [comment]
How does that look scrolled down? I.e. when the examples are expanded and I want to finally write the rule? Should the rule input field always be visible and only the examples scrolling (if expanded)?

**Comment by katierik** [comment]
To make sure I understand - 
1 - We cannot achieve the spacing around the expandable header
image

2 - We cannot indent the content in
image

?

**Comment by josejulio** [comment]
> 1 - We cannot achieve the spacing around the expandable header
> 2 - We cannot indent the content in

I think we can do that, @spinder can you apply these changes?

Also, for the monospaced fonts, we can use the font-family 'monospace'.
To do that we can create a monospace className with typestyle and use `fontFamily: 'monospace'` and apply that className where required, see https://github.com/RedHatInsights/policies-ui-frontend/blob/8ae293685763bb102bb257518201a66920bb82d3/src/components/Policy/Table/ExpandedContent/Actions.tsx#L22-L24 and https://github.com/RedHatInsights/policies-ui-frontend/blob/8ae293685763bb102bb257518201a66920bb82d3/src/components/Policy/Table/ExpandedContent/Actions.tsx#L93

**Comment by katierik** [comment]
We can probably reduce the spacing more with the text styles
image

**Comment by spinder** [comment]
Screenshot2020-04-20_14-25-27

I think I've made the suggested changes, but it still a bit too big(IMO) to see the Conditions after expanding.  @josejulio, @katierik ? Wdyt.

**Comment by spinder** [comment]
Screenshot2020-04-20_17-28-51
Indented the usage area.

File: src/components/Policy/Table/ExpandedContent/Usage.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+const codeClass = style({
+    fontFamily: 'monospace',
+    fontSize: '14px',
+    fontStyle: 'default'
+});
+const linkMarginClassName = style({
+    marginBottom: 5
+});
+const padLeftClassName = style({
+    marginLeft: 12
```
Comment (josejulio) [inline src/components/Policy/Table/ExpandedContent/Usage.tsx:15]: Could you bump this to 20?

**Comment by josejulio** [comment]
With the typeahead on conditions, the options are hard to use / see when "condition examples" are unrolled.
When I try to scroll, it loses it focus and the suggestions box closes.

Should we move the condition text to the top? or what we can do in this case?

Peek 2020-04-21 11-41
@katierik

**Comment by spinder** [comment]
Screenshot_2020-04-21_14-10-58
Screenshot_2020-04-21_14-10-28
Screenshot2020-04-21_14-09-05
Screenshot_2020-04-21_14-08-14
@josejulio, does this work? I did some quick checking and it looks like the widgets continue to work correctly. Please give it the once over..

**Pull request merged**